### PR TITLE
Added Pset_StationEquation

### PR DESCRIPTION
**Pull request by larswik**
Pset_LinearReferencingMethod remains...

### CODE REVIEW

File: IFC4x3/Properties/i/IncomingStation_2ScYZDmQ9C9vR878Q0zliC/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The station value for the incoming segment that ends at this referent location.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/IncomingStation_2ScYZDmQ9C9vR878Q0zliC/Documentation.md:1]: Could probably delete 'referent' here as it is an abstract concept.

File: IFC4x3/Properties/o/OutgoingLabel_2q56pakCnBwBBrSMnp550b/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The label referencing the outgoing segment that starts at this referent location. If omitted, OutgoingLabel value is equal to the IncomingLabel value.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/o/OutgoingLabel_2q56pakCnBwBBrSMnp550b/Documentation.md:1]: Simplification proposal 'The label of the outgoing segment...'.

File: IFC4x3/Properties/o/OutgoingStation_2wUFXsqVDB8fwdTk7zGJXN/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The station value for the outgoing segment that starts at this referent location. If omitted, OutgoingStation value is equal to IncomingStation value.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/o/OutgoingStation_2wUFXsqVDB8fwdTk7zGJXN/Documentation.md:1]: Article correction 'station value of'.

**Review by SergejMuhic — COMMENTED** [review]
Thanks Lars. If you do not agree with the proposals, just let me know and I will merge.

File: IFC4x3/Properties/i/IncomingStation_2ScYZDmQ9C9vR878Q0zliC/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/i/IncomingStation_2ScYZDmQ9C9vR878Q0zliC/Documentation.md:1]: Fixing, and also changing replacing "for" with "of" as with OutgoingStation

File: IFC4x3/Properties/o/OutgoingStation_2wUFXsqVDB8fwdTk7zGJXN/Documentation.md (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/o/OutgoingStation_2wUFXsqVDB8fwdTk7zGJXN/Documentation.md:1]: Fixing. Also removing "referent".

**Pull request merged**